src/main/java/com/onegini/examples/resourcegateway/model/ErrorResponse.java

```diff
@@ -2,21 +2,14 @@
 
 import java.io.Serializable;
 
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+@Data
+@RequiredArgsConstructor
```

Comment:
nit; The `RequiredArgsConstructor` is ensured by the `Data` and can be removed.

Reply:
Ohh, I didn't know that 👍